### PR TITLE
[Term Entry] Lua Math .fmod()

### LINKED ISSUE #3394: [Term Entry] Lua Math .fmod()
### Reason/inspiration (optional)

We would like a new term entry on the `.fmod()` method for Lua: [Math](https://www.codecademy.com/resources/docs/lua/mathematical-library). The entry should go in a new file under `docs/content/lua/concepts/mathematical-library/terms/fmod/fmod.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```lua `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Add term **fmod.md** to Lua Math. Closes #3394 .

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: '.fmod()'
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:2]:
```suggestion
Title: 'fmod()'
```

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Description: 'Returns the remainder of the quotient of two parameters.'
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:3]:
```suggestion
Description: 'Returns the remainder of the quotient of two arguments.'
```

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The method **`.fmod()`** takes two numbers as parameters and returns the remainder of the first number divided by the second number.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:16]:
```suggestion
The method **`fmod()`** takes two numbers and returns the remainder of the first number divided by the second number.
```

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Syntax
+
+The syntax of `.fmod()` is as follows:
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:20]:
```suggestion
The syntax of `fmod()` is as follows:
```

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+```pseudo
+math.fmod(num1, num2)
+```
+
+Where `num1` names the number to be divided by `num2`.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:26]:
```suggestion
Where `num1` is the value to be divided by `num2`.
```

File: content/lua/concepts/mathematical-library/terms/fmod/fmod.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+## Examples
+
+The following examples illustrate the behavior of `.fmod()`:
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/fmod/fmod.md:30]:
```suggestion
The following examples illustrate the behavior of `fmod()`:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@noahpgordon I've added my initial comments. Please update the entry, and tag me when you've completed the changes. Thanks

**Comment by noahpgordon** [comment]
@caupolicandiaz I've made the above edits.

**Review by caupolicandiaz — APPROVED** [review]
@noahpgordon This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon Looks good.

**Pull request merged**